### PR TITLE
Update ros workspace script and instructions.

### PR DESCRIPTION
**Pull request by Brian-Acosta**
This change is [Reviewable](https://reviewable.io/reviews/DAIRLab/dairlib/340)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,12 +64,11 @@ source /opt/ros/noetic/setup.bash
 ```
 3. Install additional dependencies
 ```
-sudo apt install python-rosinstall-generator python-catkin-tools
+sudo apt install python-rosinstall-generator python-catkin-tools python3-vcstool
```
Comment (ebianchi) [inline README.md:67]: The `python-rosinstall-generator` for us also had to change to `python3-rosinstall-generator`.

**Comment by ebianchi** [comment]
The instructions should also have the `install_prereqs_focal.sh` (or the Mac or other Ubuntu options) somewhere in them.

**Review by ebianchi — CHANGES_REQUESTED** [review]
A couple changes requested:

- [ ] The first is one of the other packages needed its `python` prefixed to be updated to `python3` for us.  See my comment on the specific line of code where that showed up.
- [ ] Unless I missed a call to it somewhere in one of the other installation scripts, nowhere in these instructions is a call to `install_prereqs_XXX.sh`.  Wherever appropriate, it would be useful to include this step in the instructions.

**Review by Brian-Acosta — COMMENTED** [review]
1. Done
2. Listed as option 1a of the **Build Drake** header 

> *[Reviewable](https://reviewable.io/reviews/DAIRLab/dairlib/340#-:-NPtPlEx0VD4JSYyQk6R:b-yv2fs0)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @ebianchi)

___
*[`README.md` line 67 at r1](https://reviewable.io/reviews/DAIRLab/dairlib/340#-NPchk10olNP_WauGOwx-r1-67:-NPtPaaU53oaNF9Y73aC:bb6un1j) ([raw file](https://github.com/DAIRLab/dairlib/blob/6efc7466124ed2ad6c1be360b8e76d15434d7ba0/README.md#L67)):*
<details><summary><i>Previously, ebianchi (Bibit Bianchini) wrote…</i></summary><blockquote>

The `python-rosinstall-generator` for us also had to change to `python3-rosinstall-generator`.
</blockquote></details>

Done

**Comment by ebianchi** [comment]
LGTM

**Review by ebianchi — APPROVED** [review]

**Pull request merged**